### DISCUSSION TITLE
Cant create a TypeSupport

### DISCUSSION THREAD

**Discussion by AlbertoElDevaluador** [jmmorato/openddsharp#195] [Q&A]
I cant create a TypeSupport with a IDL having a module with this name
`
module zkey
{
}
`
Do you know what happend??

**Comment by jmmorato** [comment]
I don't see any problem with the module name you used, I would say it is something related with the rest of your IDL, perhaps you used the same name for a structure or field in the same IDL? 

It works for me with the following code:

```
module zkey {
    struct test {
        string a;
    };
};
```
